.github/workflows/release-ios.yml

```
@@ -0,0 +1,150 @@
+      - name: Setup Rust
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          targets: 'aarch64-apple-ios, x86_64-apple-ios, aarch64-apple-ios-sim'
+      - name: Retrieve cache
+        uses: Leafwing-Studios/cargo-cache@v2
+      - name: Install the Apple certificate
+        env:
+          BUILD_CERTIFICATE_BASE64: ${{ secrets.IOS_BUILD_CERTIFICATE_BASE64 }}
+          P12_PASSWORD: ${{ secrets.IOS_P12_PASSWORD }}
+          KEYCHAIN_PASSWORD: ${{ secrets.IOS_KEYCHAIN_PASSWORD }}
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:6:58: Double quote to prevent globbing and word splitting

(shellcheck)